NEWS.md

```diff
@@ -8,6 +8,8 @@
 
 * `coord_map()` now can have axes on the top and right (@karawoo, #3042).
 
+* Clearer error messages for innappropriate aesthetics. (@clairemcwhite, #3060).
```

Comment:
There shouldn't be a period after "aesthetics" (only at the very end).

Reply:
Fixed. Also fixed wrong spelling of inappropriate.